File: tests/pyconverter-test/cases/class_static_local.py
class Greeter:
    greeting = "default"
    def __init__(self, message):

        print ("constructor: " + self.greeting)
        self.greeting = message
        self.local = "local"




greeter = Greeter("world")


print (greeter.greeting)
print (Greeter.greeting)

Greeter.greeting = "newdefault"
greeter = Greeter("world")
print (Greeter.greeting)
greeter.greeting = None
print (greeter.greeting)

File: tests/pyconverter-test/baselines/class_static_local.ts
class Greeter {
    static greeting: string
    private ___greeting_is_set: boolean
    private ___greeting: string
    get greeting(): string {
        return this.___greeting_is_set ? this.___greeting : Greeter.greeting
    }
    set greeting(value: string) {
        this.___greeting_is_set = true
        this.___greeting = value
    }
    
    local: string
    public static __initGreeter() {
        Greeter.greeting = "default"
    }
    
    constructor(message: string) {
        console.log("constructor: " + this.greeting)
        this.greeting = message
        this.local = "local"
    }
    
}

Greeter.__initGreeter()

let greeter = new Greeter("world")
console.log(greeter.greeting)
console.log(Greeter.greeting)
Greeter.greeting = "newdefault"
greeter = new Greeter("world")
console.log(Greeter.greeting)
greeter.greeting = null
console.log(greeter.greeting)
